src/ui/DynamicHierarchicalFacet/DynamicHierarchicalFacet.ts

```diff
@@ -206,6 +207,19 @@ export class DynamicHierarchicalFacet extends Component implements IDynamicHiera
      * **Default:** `true` if folded results are requested;`false` otherwise.
      */
     filterFacetCount: ComponentOptions.buildBooleanOption({ section: 'Filtering' }),
+
+    /**
+     * The sort criterion to use for this facet.
+     *
+     * Allowed values are `occurences` and `alphanumeric`.
+     *
+     * **Default (API):** `occurrences`
+     */
+    sortCriteria: <FacetSortCriteria>ComponentOptions.buildStringOption({
+      postProcessing: value => (value === FacetSortCriteria.alphanumeric || value === FacetSortCriteria.occurrences ? value : undefined),
```

Comment:
Should probably add some kind of warning for the user ? 

That explains that an invalid criteria was detected, and the list of valid/supported criteria.